### ISSUE TITLE
Re-enrollment of aCM agents using an Enrollment guide

### ISSUE THREAD

**Issue by Calebasah** [ADORSYS-GIS/wazuh-agent#17]
Link to [#94](https://github.com/ADORSYS-GIS/wazuh/issues/94)
1. Document a comprehensive enrollment guide to for prod cluster --> @Calebasah - [x] Create enrollment guide 
- [x] Document all the process 

2. Enrolling teams in cameroon:
- [x] Online Banking
- [x] Stripe Team
- [x] Rust Team
- [x] Datasafe Team
- [x] Solution Team
- [x] DATEV

https://adorsys.atlassian.net/wiki/spaces/Wazuh/database/1413055815

**Comment by yannicksiewe** [comment]
WIP by @Calebasah -> Please check Jude Wazuh agent setting

**Comment by Calebasah** [comment]
**Observations:**
**1. Disconnected Agents in the Morning:**
- Some Wazuh agents are disconnected despite the service running.
- The Wazuh components `(sudo /var/ossec/bin/wazuh-control status)` are not functioning until restarted manually.

**2. API Response Issues:**
- When requesting the stats or data inventory of an agent , the request sometimes fails, even though it works at other times.
- This is accompanied by an Axios timeout error `(timeout of 20000ms exceeded)`, indicating the server is not responding within the specified time limit.

**Comment by MarantosGeorge** [comment]
On hold due to unstable prod cluster

**Comment by ArmandMeppa** [comment]
Due to new updates, we will have to clean the cluster and re-enrol all the agents back

**Comment by yannicksiewe** [comment]
WIP, could happen that we have to re-enroll all the agent

**Comment by yannicksiewe** [comment]
Cameroon team will be re-enrolled starting today 24.01.25
-> Upgrade server 4.10.1
-> Agent version 4.10.1

**Comment by MarantosGeorge** [comment]
Waiting on server upgrade to 4.10.1 to start.

**Comment by ArmandMeppa** [comment]
Will start today with the **Stripe** and **Marketing** teams

**Comment by yannicksiewe** [comment]
WIP

**Comment by yannicksiewe** [comment]
Some users are having issue with Snort v2.9 on Linux Ubuntu v22.04
@Calebasah  is working on this

**Comment by Calebasah** [comment]
WIP

**Comment by MarantosGeorge** [comment]
Done. Changing OS from 23 to 24.04 for 2 people from Solution Team in order to enroll.
